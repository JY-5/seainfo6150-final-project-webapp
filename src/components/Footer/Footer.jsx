import React from 'react'
import styles from './Footer.module.css';

const Footer = () => {
    return (
        <div className={styles.container}>
            Contact us: (408) 123-4567
        </div>
    )
}

export default Footer
