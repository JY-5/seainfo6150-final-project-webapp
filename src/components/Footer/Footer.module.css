.container {
    height: 1rem;
    padding: 1rem;
    color: rgb(139, 136, 136);
    background-color: #fbfbfb;
}