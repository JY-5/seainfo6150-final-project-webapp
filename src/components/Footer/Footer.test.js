import React from "react";
import Footer from './Footer';
import { render } from "@testing-library/react";

describe("Add Footer tests", () => {
    it("renders correctly", () => {
      const { container } = render(<Footer/>);
      expect(container).toMatchSnapshot();
    });
});