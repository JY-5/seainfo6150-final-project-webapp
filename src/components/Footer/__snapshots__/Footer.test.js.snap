// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Footer tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    Contact us: (408) 123-4567
  </div>
</div>
`;
